src/katello/enabled_report.py

```diff
@@ -1,10 +1,10 @@
 import os.path
 from ConfigParser import ConfigParser
```

Comment:
Replacing this line with `from katello.utils import ConfigParser` should address the test failures :)